roles/system/tasks/tools.yml

```diff
@@ -24,3 +24,11 @@
       - imagemagick
       - ruby-dev
       - shared-mime-info
+
+- name: Install policykit-1
+  become: true
+  apt:
+    state: latest
+    update_cache: true
+    name: policykit-1
+  when: ansible_os_family == 'Debian'
```

Comment:
AFAIK Ubuntu systems belong to the Debian OS family, so this condition is always true in our case :thinking:. Maybe we can add `policykit-1` to the list of tools above instead?